src/Toggl2Vertec/Toggl/TogglAccessCheck.cs

```diff
@@ -15,11 +15,11 @@ public TogglAccessCheck(TogglClient client)
 
         public override bool Check(ICliLogger logger)
         {
-            logger.LogPartial(logger.CreateText($"Checking Toggl API access (https://api.track.toggl.com/api/v8/me): "));
+            logger.LogPartial(logger.CreateText("Checking Toggl API access (https://api.track.toggl.com/api/v9/me): "));
             try
             {
                 var profile = _client.FetchProfileDetails();
-                if (profile.GetProperty("data").GetProperty("id").GetInt32() <= 0)
+                if (profile.GetProperty("id").GetInt32() <= 0)
```

Comment:
I see. _Massive_ restructuring of the data structure. I'm sure that was worth it for Toggl 😝

Reply:
version nine! it's... bigger than eight! 12.5% increase in API version! just imagine the exponential potential of a 12.5% growth rate!! in two years time, the whole universe could be paper clips and we'd _still_ be increasing the version!!!